### PR TITLE
docs: alerting: clarify which panels support alerts

### LINKED ISSUE #87909: Alert tab not visible for panels for certain visualization types
### What happened?

- Create a new panel with "time series" type
- Add a query, save the panel
- Add an alert to the panel (using the "Alert" tab on the panel editor) and save it
- Check appearance: the panel shows on the dashboard, and also the heart icon is showing the state of the associated alert
- Then go back and edit the panel
- Change its visualization type to something else - in my case I changed it to "heatmap"
- Notice that the "Alerts" tab disappears from the panel editor <<<< this is the actual bug
- Save the panel and look at the dashboard. The alert heart icon is still there ,and the alert is still associated with the panel. It works, but it is not accessible through the panel editor.

image

image

If you change back the visualization type to time series, then the alert re-appears.

### What did you expect to happen?

Always show the "Alert" tab, independently of the panel type. The alert can have a separate underlying database query; there is no reason to hide it.

### Did this work before?

I have no idea if it was working before.

### How do we reproduce it?

Please see the "what happened" section for this.

### Is the bug inside a dashboard panel?

I'm not able to paste the panel's JSON here because the db query is confidental, but this problem is a general, so it seems.

### Environment (with versions)?

Grafana: [Grafana v11.0.0 (277ef258d4)](https://github.com/grafana/grafana/blob/main/CHANGELOG.md)
OS: Server is Ubuntu 22.04.3 LTS
Browser: Chrome on linux (not relevant, it happens on all browsers)

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

### Datasource(s)?

_No response_

### PR DESCRIPTION
**Pull request by tonypowa**
**What is this feature?**

There was no indication in the docs regarding which visualizations support alerts
This question has come up a few times .e.g https://github.com/grafana/grafana/issues/87909

Fixes #87909

### CODE REVIEW

**Comment by tonypowa** [comment]
tested locally. links and note boxes work 
@brendamuir @srperf  let me know if the copy for the time series doc looks good enough

**Comment by tonypowa** [comment]
preview of modified files:

image



image

image

File: docs/sources/alerting/alerting-rules/create-grafana-managed-rule.md [hunk 1]
```diff
@@ -227,6 +237,10 @@ Annotations add metadata to provide more information on the alert in your alert
 
    Links alerts to panels in a dashboard.
 
+   {{% admonition type="note" %}}
+   At the moment, alerts are only supported in the [time series](ref:time-series), and [alert list](ref:alert-list) visualizations.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-grafana-managed-rule.md:241]:
```suggestion
   At the moment, alerts are only supported in the [time series](ref:time-series) and [alert list](ref:alert-list) visualizations.
```

File: docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md [hunk 2]
```diff
@@ -134,6 +144,10 @@ Annotations add metadata to provide more information on the alert in your alert
 
    Links alerts to panels in a dashboard.
 
+   {{% admonition type="note" %}}
+   At the moment, alerts are only supported in the [time series](ref:time-series), and [alert list](ref:alert-list) visualizations.
```
Comment (brendamuir) [inline docs/sources/alerting/alerting-rules/create-mimir-loki-managed-rule.md:148]:
```suggestion
   At the moment, alerts are only supported in the [time series](ref:time-series) and [alert list](ref:alert-list) visualizations.
```

File: docs/sources/panels-visualizations/visualizations/_index.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ If you are unsure which visualization to pick, Grafana can provide visualization
 {{< /admonition >}}
 
 - Graphs & charts
-  - [Time series](ref:time-series) is the default and main Graph visualization.
+  - [Time series](ref:time-series) is the default and main Graph visualization. Alerts are supported in this panel.
```
Comment (brendamuir) [inline docs/sources/panels-visualizations/visualizations/_index.md:165]:
```suggestion
  - [Time series](ref:time-series) is the default and main graph visualization. Alerts are supported in this panel.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 4]
```diff
@@ -65,6 +75,11 @@ refs:
 
 Time series visualizations are the default and primary way to visualize time series data as a graph. They can render series as lines, points, or bars. They're versatile enough to display almost any time-series data.
 
+{{% admonition type="note" %}}
+You can [link alert rules](ref:link-alert) to the time series visualization to observe when alerts fire and get resolved in the form of annotations. In addition, you can create alert rules from the Alert tab within the panel options.
```
Comment (brendamuir) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:79]:
```suggestion
You can [link alert rules](ref:link-alert) to time series visualization to observe when alerts fire and are resolved in the form of annotations. In addition, you can create alert rules from the Alert tab within the panel options.
```

File: docs/sources/panels-visualizations/visualizations/time-series/index.md [hunk 5]
```diff
@@ -65,6 +75,11 @@ refs:
...
+At the moment, alerts are only supported in the time series, and [alert list](ref:alert-list) visualizations.
```
Comment (brendamuir) [inline docs/sources/panels-visualizations/visualizations/time-series/index.md:80]:
```suggestion
At the moment, alerts are only supported in the time series and [alert list](ref:alert-list) visualizations.
```

**Review by brendamuir — APPROVED** [review]

**Review by srperf — APPROVED** [review]

**Comment by imatwawana** [comment]
@tonypowa - Does this still need to be backported to v11.0 or no?

**Comment by tonypowa** [comment]
ops i think i accidentally closed the backport PR. I just reopened and merged it. 
Thanks for letting me know!

**Pull request merged**